lte/gateway/python/magma/enodebd/data_models/data_model_parameters.py

```diff
@@ -63,6 +61,8 @@ class ParameterName():
     NUM_PLMNS = 'Num PLMNs'
     PLMN = 'PLMN'
     PLMN_LIST = 'PLMN List'
+    MME_POOL_1 = 'MME Pool 1'
```

Comment:
For how to resolve this lint, see https://wemake-python-stylegui.de/en/latest/pages/usage/violations/naming.html#wemake_python_styleguide.violations.naming.UnderscoredNumberNameViolation